pkg/registry/scheduling/rest/storage_scheduling.go

```diff
@@ -90,7 +90,8 @@ func AddSystemPriorityClasses() genericapiserver.PostStartHookFunc {
 				_, err := schedClientSet.PriorityClasses().Get(context.TODO(), pc.Name, metav1.GetOptions{})
```

Comment:
How about `pc.Name`, would it cause data race?

Reply:
pc.Name is ok. the root cause can refer to this issue https://github.com/kubernetes/kubernetes/issues/82497